tdms/include/iterator.h

```diff
@@ -13,6 +13,12 @@
 #include "field.h"
 #include "grid_labels.h"
 
+/**
+ * @brief The Mother of all Functions
+ * Used to be the MATLAB mexFunction
+ */
+void moaf(int, mxArray**, int, const mxArray**, DerivativeMethod);

```

Comment:
I guess "execute_simulation" would be a reasonable name? It depends what we do as we refactor `iterator.cpp` #63 to be honest. 

My vote is still for "Lord of all functions (loaf)".

Reply:
I went with `execute_simulation`.